### ISSUE TITLE
Invalid option in Nanopore-UL-Jun2020 and Nanopore-Jun2020

### ISSUE THREAD

**Issue by LYC-vio** [chanzuckerberg/shasta#290]
When running
```
./shasta-Linux-0.10.0 --command listConfiguration --config Nanopore-Jun2020
```
or
```
./shasta-Linux-0.10.0 --command listConfiguration --config Nanopore-UL-Jun2020
```
Shasta returns:
```
Invalid option: unrecognised option 'MarkerGraph.refineThreshold'
```

**Comment by paoloczi** [comment]
Yes, we generally don't guarantee that all assembly configurations continue to work with all Shasta versions. That options has since been removed. This was announced in the release notes for Shasta 0.9.0 (in the section on deprecated features).

Depending on your situation, you have a few options:

- If you have new reads (Guppy 5 or newer base caller with "super" accuracy) use the newer and more accurate `--config Nanopore-May2022`.
- Switch to an older Shasta release.
- If you really want to use the `Nanopore-Jun2020` assembly configuration with Shasta 0.10.0, use a modified version of configuration file `shasta/conf/Nanopore-Jun2020.conf` (you will have to download that from GitHub). You will have to edit the file to remove the offending option, which is no longer supported in Shasta 0.10.0.

But you have a valid point that we should not have built-in configurations that don't work. For future releases I will make sure to remove any built-in assembly configurations that are no longer functional. However the `shasta/conf` directory wil continue to contain older assembly configurations that are no longer supported, for completeness and for reference.

Please let me know if I can help better regarding the selection of an assembly configuration appropriate for your situation.

**Comment by LYC-vio** [comment]
Thank you very much for your quick reply! I'll switch to Nanopore-UL-Dec2019 then.

Thanks again!

**Comment by paoloczi** [comment]
I am reopening this as a reminder that we need to add a test to check that all built-in assembly configurations are functional, and remove the ones that are not.

Thank you for bringing this up.

**Comment by paoloczi** [comment]
In commit 7f824e1b91125bd89245a7de342d6559fc418ced I added `shasta/scripts/CheckConfigurations.py` which checks that all built-in configuration work (two of them don't and will removed in a subsequent commit). This script will be run every time before making a new release.
